modules/nft/src/lib.rs

```diff
@@ -52,22 +56,6 @@ pub mod weights;
 pub use module::*;
 pub use weights::WeightInfo;
 
-pub type CID = Vec<u8>;
-pub type Attributes = BTreeMap<Vec<u8>, Vec<u8>>;
-
-#[repr(u8)]
-#[derive(Encode, Decode, Clone, Copy, BitFlags, RuntimeDebug, PartialEq, Eq, TypeInfo)]
-pub enum ClassProperty {
-	/// Is token transferable
-	Transferable = 0b00000001,
-	/// Is token burnable
-	Burnable = 0b00000010,
-	/// Is minting new tokens allowed
-	Mintable = 0b00000100,
-	/// Is class properties mutable
-	ClassPropertiesMutable = 0b00001000,
-}
-
 #[derive(Clone, Copy, PartialEq, Default, RuntimeDebug)]
 #[cfg_attr(feature = "std", derive(Serialize, Deserialize))]
 pub struct Properties(pub BitFlags<ClassProperty>);
```

Comment:
Make a new type, so `enumflags2` can be removed from this module.

Reply:
Moved Properties to Primitives too, so no need to enumflag2 in the nft module